### DISCUSSION TITLE
Using a different certificate for the HTTPS server

### DISCUSSION THREAD

**Discussion by thgeorgiou** [goauthentik/authentik#1400] [Q&A] [answered]
Hello!

Is it possible to use a different certificate for the integrated HTTPS server (ie. the UI)?

Thanks a lot, authentik seems great so far!

**Comment by BeryJu** [comment, marked as answer]
Hi, currently thats not possible, nor intended really, I'm assuming most people will have an existing reverse-proxy already setup, that deals with their SSL Termination etc.

**Reply by thgeorgiou** [reply]
Thanks for your reply! I have an internal CA that I use to generate certs for all services so internal communications are encrypted (ie. from the reverse proxy to authentik). Not a big deal to workaround.
